apps/admin-gui/src/app/shared/components/dialogs/create-service-dialog/create-service-dialog.component.html

```diff
@@ -0,0 +1,72 @@
+<h1 mat-dialog-title>{{'DIALOGS.CREATE_SERVICE.TITLE' | translate}}</h1>
+<div class="{{theme}}">
+  <div mat-dialog-content class="dialog-container">
+    <mat-form-field>
+      <input
+        matInput
+        required
+        (change)= "makePath()"
+        [formControl]="nameControl"
+        placeholder="{{'DIALOGS.CREATE_SERVICE.NAME' | translate}}"
+      >
+      <mat-error>{{'DIALOGS.CREATE_SERVICE.REQUIRE_NAME' | translate}}</mat-error>
+    </mat-form-field>
+
+    <mat-form-field>
+      <input
+        [(ngModel)]="description"
+        matInput
+        placeholder="{{'DIALOGS.CREATE_SERVICE.DESCRIPTION' | translate}}"
+      >
+    </mat-form-field>
+
+    <mat-form-field>
+      <input
+        matInput
+        [formControl]="delayControl"
+        placeholder="{{'DIALOGS.CREATE_SERVICE.DELAY' | translate}}"
+      >
+      <mat-error>{{'DIALOGS.CREATE_SERVICE.INVALID_DELAY' | translate}}</mat-error>
```

Comment:
Here you probably wanted to say that this field is required, but forget to place here "required", because now you make it possible that this field is empty.